src/utils/helpers.tsx

```diff
@@ -0,0 +1,23 @@
+import { Filter } from '../types/Filter';
+import { Todo } from '../types/Todo';
+
+export const prepareTodos = (
+  todosList: Todo[],
+  selectedFilter: Filter,
+): Todo[] => {
+  let filteredTodos = [...todosList];
+
+  switch (selectedFilter) {
+    case Filter.Active:
+      filteredTodos = todosList.filter(todo => !todo.completed);
+      break;
+
+    case Filter.Completed:
+      filteredTodos = todosList.filter(todo => todo.completed);
+      break;
+    default:
+      break;
```

Comment:
Do not leave the `default` block empty, otherwise what is the point of using a `switch` structure